OutOfSchool/OutOfSchool.WebApi/Services/WorkshopServicesCombiner.cs

```diff
@@ -185,6 +213,18 @@ private bool IsFilterValid(WorkshopFilter filter)
             return filter != null && filter.MaxStartTime >= filter.MinStartTime
                                   && filter.MaxAge >= filter.MinAge
                                   && filter.MaxPrice >= filter.MinPrice;
+        } 
+        
+        public async Task<IEnumerable<string>> GetNotificationsRecipientIds(NotificationAction action, Dictionary<string, string> additionalData, Guid objectId)
```

Comment:
In the story mentioned that 
> all users that liked the workshop and applied see notifications about the change. 

I'm not sure what does _**applied users**_ mean and why we should notify them. It is better to ask BA.